### DISCUSSION TITLE
Next CP2K Dev Meeting

### DISCUSSION THREAD

**Discussion by robertschade** [cp2k/cp2k#2853] [General]
Dear @cp2k/cp2k-core-developers  @cp2k/cp2k-developers,
we would like to schedule our next CP2K Dev meeting (see also [https://www.cp2k.org/dev:meetings](https://www.cp2k.org/dev:meetings)) soon. First, I would like to find out which weeks to exclude in the first place because of vacations and other things so that we can limit the questions for the actual slots. The dev meeting is usually a 2-hour slot in the afternoon.
I would be happy if you could comment below if one or multiple of the options below won't work for you till end of next week (14th of August).
Many Thanks 
Robert

week 30: 24th of July-28th of July
week 31: 31st of July-4th of August
week 32: 7th of August-11th of August
week 33: 14th of August-18th of August
week 34: 21st of August-25th of August

PS: If someone knows how to have a multiple-selection pool on GitHub discussions, please let me know.

**Comment by mkrack** [comment]
@robertschade week 30 and first part half of week 31 do not work for me.

**Comment by oschuett** [comment]
I will be unavailable in week 30.

**Comment by hfp** [comment]
week 33 and week 34 work for me.

**Comment by robertschade** [comment]
Hi @cp2k/cp2k-core-developers  @cp2k/cp2k-developers,
please enter your availability for the next CP2K-Developers meeting in https://terminplaner6.dfn.de/de/p/3bfb5f87a330294f1c093cd646d86a63-333537
till 28th of July.
Please feel free to add topics, questions, or slides to the meeting notes at
https://docs.google.com/presentation/d/1ZblD6WNEJEV4EsfJOmpvMvC_Nxzh0-62ePolLYuR3n4/edit?usp=sharing
beforehand. (Because this post is publically visible, I can't post here the edit-link for the slides, but simply comment or send me a mail, then I will enable edit access for you. You can also send me the slides or topics/questions that you would like to have added there.)

You can also find notes from previous meetings at https://www.cp2k.org/dev:meetings.

**Reply by robertschade** [reply]
Dear @cp2k/cp2k-developers @cp2k/cp2k-core-developers ,
we would like to invite you to the next CP2K developers Meeting on August 18th, 14:00-16:00 CEST in
https://uni-paderborn-de.zoom.us/j/94563757798?pwd=bXcyaEFpR0VSN0FjQzIxMWM2USszdz09
Please feel free to add topics, questions, or slides to the meeting notes at
https://docs.google.com/presentation/d/1ZblD6WNEJEV4EsfJOmpvMvC_Nxzh0-62ePolLYuR3n4/edit?usp=sharing
beforehand or comment here.
(Because this post is publically visible, I can't post here the edit-link for the slides, but simply comment or send me a mail, then I will enable edit access for you. You can also send me the slides or topics/questions that you would like to have added there.)

You can also find notes from previous meetings at https://www.cp2k.org/dev:meetings.
